integrations/src/test/resources/io/jenkins/plugins/casc/SbtConfiguratorTest.yml

```diff
@@ -0,0 +1,10 @@
+tool:
```

Comment:
Ideally this test would go to the plugin, and then documented on the plugin docs and a demo added here